pymc/step_methods/pgbart.py

```diff
@@ -274,6 +259,19 @@ def init_particles(self, tree_id):
 
         return np.array(particles)
 
+    def update_weight(self, particle):
+        """
+        Update the weight of a particle
+
+        Since the prior is used as the proposal,the weights are updated additively as the ratio of
+        the new and old log_likelihoods.
+        """
```

Comment:
Can you add a type hint to the signature?
```suggestion
    def update_weight(self, particle):
        """
        Update the weight of a particle

        Since the prior is used as the proposal, the weights are updated additively as the ratio of
        the new and old log-likelihoods.
        """
```